arcade/__init__.py

```diff
@@ -34,6 +35,50 @@ def configure_logging(level: Optional[int] = None):
         LOG.addHandler(ch)
 
 
+def get_timestamp(
+        how: str = "%Y_%m_%d_%H%M_%S_%f%Z",
+        when: Optional[types.HasStrftime | datetime] = None,
```

Comment:
1. Did `HasStrftime` not match `datetime`? Was it only pyright which complained?
2. To my understanding, the `|` syntax is a 3.9+ feature, but we support 3.8+

Reply:
No it didn't.  pyright complained and the build failed.

Reply:
Are you sure? I see references to line 41 in the GitHub CI builds on the previous commits. That's the other lines it seems.